accumulate.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudscale-ch/cloudscale-go-sdk/v2"
+	"os"
+	"time"
+)
+
+// Represents one data point ("fact") in the billing database. The actual value for the data point is not present in
+// this type, as this type is just a map key, and the corresponding value is stored as a map value.
+type AccumulateKey struct {
+	Query     string
+	Zone      string
+	Tenant    string
+	Namespace string
+	Start     time.Time
+}
```

Comment:
FYI: GoDoc follows the convention of `<type-or-func-name> <verb> <object>`

Nitpick: I usually follow our one-line-per-sentence that we have for AsciiDoc also for code comments. IMO sentences are easier to maintain when they're in a single line rather than broken into multiple ones due to the old-school 80-char limit style, wide-screen monitors are pretty common nowadays. But that's a personal opinion, no hard feeling.
```suggestion
// AccumulateKey represents one data point ("fact") in the billing database.
// The actual value for the data point is not present in this type, as this type is just a map key, and the corresponding value is stored as a map value.
type AccumulateKey struct {
	Query     string
	Zone      string
	Tenant    string
	Namespace string
	Start     time.Time
}
```